log.sh

```diff
@@ -76,9 +76,16 @@ LSLOG () {
   # if no message was passed, read it from STDIN
   local _MSG
   [[ $# -ne 0 ]] && _MSG="$@" || _MSG="$(cat)"
-  echo -ne "$_LS_LEVEL_BEGIN$OUTPUT " >> "$LS_OUTPUT"
-  echo -n  "$_MSG"                    >> "$LS_OUTPUT"
-  echo -e "$_LS_LEVEL_END" >> "$LS_OUTPUT"
+  if [[ "$LS_OUTPUT" = "/dev/stdout" ]]
+  then
```

Comment:
I'm using the github.com web interface, and I didn't see any --amend style options.

This guide says you should be able to flatten all commits in my branch into a single commit on merge, though:
https://github.com/blog/2141-squash-your-commits

If you're using the CLI, though, I don't have a clue how you might do that.  If that's the case, I'll just make another fork/edit/PR.